docs/source/deployguide/cadeploy.md

```diff
@@ -0,0 +1,597 @@
+# CA Deployment steps
+
+## Download the binaries
+
+The Fabric CA server and CA client binaries are available for download from [github](https://github.com/hyperledger/fabric-ca/releases). Scroll down to **Assets** and select the latest binary for your machine. The .zip file contains both the CA server and the CA client binaries. Copy the CA server file to a directory where you will run the server. Likewise, copy the CA client binary to its own directory. Having the CA client in its own folder facilitates certificate management especially when you need to interact with multiple CAs.  When you issue a command from the CA client against a CA server, you can target a specific CA by modifying the CA server URL on the request. Therefore, only a single Fabric CA client binary is required and can be used to transact with multiple CAs. More on using the Fabric CA client below.
```

Comment:
While I agree with the 'binary first' approach, I think it is worth at least mentioning that you can also choose to run the Fabric CA image.

Reply:
This is what I settled on adding:

After you have mastered the use of these binaries, it is likely you will want to use the Fabric CA image for example in a Kubernetes or Docker deployment. For now though, the purpose of this topic is to teach you how to properly use the binaries.